src/shared/ui/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react"
+import './SearchForm.css'
+
+export const SearchForm = ({ initialValue = "", onSearch }) => {
+
+  const [searchText, setSearchText] = useState(initialValue);
+
+  const handleInputChange = (event) => {
+    setSearchText(event.target.value);
+  };
+
+  const handleSearchClick = () => {
+    onSearch(searchText);
+  };
+
+  const handleKeyPress=(event)=>{
+    if (event.key === "Enter") {
+      onSearch(searchText);
+    }
+  }
+
+  return (
+    <form class="searchform cf">
```

Comment:
what is `cf`, names should be understandable

Reply:
changed to understandable name